napari/plugins/io.py

```diff
@@ -100,13 +100,20 @@ def read_data_with_plugins(
             raise ValueError(message)
 
         if plugin not in plugin_manager.plugins:
-            names = {i.plugin_name for i in hook_caller.get_hookimpls()}
+            names = set(_npe2.get_readers().keys()).union(
+                {i.plugin_name for i in hook_caller.get_hookimpls()}
+            )
+            err_helper = (
+                "No readers are available. Do you have any plugins installed?"
+                if not names
+                else f"\nNames of plugins offering readers are: {names}."
+            )
```

Comment:
We always have `napari_builtins` present. So maybe we should do `if len(names) < 1`?

Reply:
Should those strings be in `trans._` calls ?

Reply:
Addressed both of these.